### PR TITLE
Message reply button

### PR DESCRIPTION
**Pull request by Valodim**
Inspired by gmail app, I moved the reply button next to the sender, to where the "flag" star was before. But compared to the reply button it doesn't deserve a highly prominent position like that, maybe we can place it somewhere else?

image

### CODE REVIEW

**Comment by cketti** [comment]
Looks nice. But won't it be annoying having to scroll up once a message has been read from top to bottom to be able to reply?

> Inspired by gmail app, I moved the reply button next to the sender, to where the "flag" star was before. But compared to the reply button it doesn't deserve a highly prominent position like that, maybe we can place it somewhere else?

Do it like the Gmail app? Always display the subject below the action bar. Put the star to the right of the subject?

**Comment by Valodim** [comment]
>  But won't it be annoying having to scroll up once a message has been read from top to bottom to be able to reply?

The "send" menu is still in overflow. Gmail covers this with a bottom bar below the message.

> Do it like the Gmail app? Always display the subject below the action bar. Put the star to the right of the subject?

Can do. It's a bit more work so I wanted to hear feedback first (and split up the PRs)

**Comment by Valodim** [comment]
image

how bout this?

**Comment by wiktor-k** [comment]
Wow, looks *very nice*, I can't wait to build it and try it out live!

**Comment by cketti** [comment]
Nice.

I'm not sure about the position of the encryption indicator. But that's easily something we can figure out later.

**Comment by Valodim** [comment]
I moved the encryption indicator so the icons form a triangle rather than a jagged pattern. There's probably still room for improvement, but overall it's certainly better than before :)

**Comment by Valodim** [comment]
And having them bigger with descriptions, too. I heard more than one person say "I can never remember which one of those buttons is for replying" on the old bottom bar. Personally I thought the arrow is the single obvious candidate, but people are reluctant to click on stuff without confirmation it's the right thing. So having big buttons with descriptions is definitely a good idea. That can be another PR, though :)

File: app/ui/src/main/java/com/fsck/k9/activity/MessageList.java [hunk 1]
```diff
@@ -946,21 +940,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
         } else if (id == R.id.delete) {
             messageViewFragment.onDelete();
             return true;
-        } else if (id == R.id.reply) {
```
Comment (cketti) [inline app/ui/src/main/java/com/fsck/k9/activity/MessageList.java]: Removing this disables the functionality in the "Send…" submenu of the action bar.

**Pull request merged**